Npm error in tabler

While installing tabler getting npm error and can't install it

<img width="759" alt="image" src="https://github.com/LaravelDaily/Larastarters/assets/64247665/ad084872-eeb4-4eb3-87ca-40827deb23a8">

`0 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'install' ]
1 info using npm@8.5.0
2 info using node@v16.14.2
3 timing npm:load:whichnode Completed in 0ms
4 timing config:load:defaults Completed in 1ms
5 timing config:load:file:/usr/local/lib/node_modules/npm/npmrc Completed in 0ms
6 timing config:load:builtin Completed in 0ms
7 timing config:load:cli Completed in 1ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:/Users/nibirahmed/Sites/nsu-app/.npmrc Completed in 1ms
10 timing config:load:project Completed in 2ms
11 timing config:load:file:/Users/nibirahmed/.npmrc Completed in 0ms
12 timing config:load:user Completed in 0ms
13 timing config:load:file:/usr/local/etc/npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:validate Completed in 0ms
16 timing config:load:credentials Completed in 1ms
17 timing config:load:setEnvs Completed in 0ms
18 timing config:load Completed in 5ms
19 timing npm:load:configload Completed in 5ms
20 timing npm:load:setTitle Completed in 5ms
21 timing config:load:flatten Completed in 1ms
22 timing npm:load:display Completed in 2ms
23 verbose logfile /Users/nibirahmed/.npm/_logs/2023-05-20T04_58_21_240Z-debug-0.log
24 timing npm:load:logFile Completed in 2ms
25 timing npm:load:timers Completed in 0ms
26 timing npm:load:configScope Completed in 0ms
27 timing npm:load Completed in 15ms
28 timing arborist:ctor Completed in 1ms
29 silly logfile start cleaning logs, removing 1 files
30 timing idealTree:init Completed in 17ms
31 timing idealTree:userRequests Completed in 0ms
32 silly idealTree buildDeps
33 silly fetch manifest @popperjs/core@^2.11.6
34 http fetch GET 200 https://registry.npmjs.org/@popperjs%2fcore 807ms (cache revalidated)
35 silly fetch manifest @tabler/core@^1.0.0-beta16
36 http fetch GET 200 https://registry.npmjs.org/@tabler%2fcore 2126ms (cache revalidated)
37 silly fetch manifest apexcharts@^3.40.0
38 http fetch GET 200 https://registry.npmjs.org/apexcharts 1133ms (cache revalidated)
39 silly fetch manifest autosize@^5.0.1
40 http fetch GET 200 https://registry.npmjs.org/autosize 978ms (cache revalidated)
41 silly fetch manifest autosize@^6.0.1
42 silly fetch manifest choices.js@^10.2.0
43 http fetch GET 200 https://registry.npmjs.org/choices.js 5ms (cache hit)
44 silly fetch manifest countup.js@^2.6.2
45 http fetch GET 200 https://registry.npmjs.org/countup.js 3ms (cache hit)
46 silly fetch manifest dropzone@^6.0.0-beta.2
47 http fetch GET 200 https://registry.npmjs.org/dropzone 3ms (cache hit)
48 silly fetch manifest flatpickr@^4.6.13
49 http fetch GET 200 https://registry.npmjs.org/flatpickr 3ms (cache hit)
50 silly fetch manifest fslightbox@^3.4.1
51 http fetch GET 200 https://registry.npmjs.org/fslightbox 4ms (cache hit)
52 silly fetch manifest imask@^6.4.3
53 http fetch GET 200 https://registry.npmjs.org/imask 3ms (cache hit)
54 silly fetch manifest jsvectormap@^1.5.3
55 http fetch GET 200 https://registry.npmjs.org/jsvectormap 3ms (cache hit)
56 silly fetch manifest list.js@^2.3.1
57 http fetch GET 200 https://registry.npmjs.org/list.js 2ms (cache hit)
58 silly fetch manifest litepicker@^2.0.12
59 http fetch GET 200 https://registry.npmjs.org/litepicker 2ms (cache hit)
60 silly fetch manifest nouislider@^15.7.0
61 http fetch GET 200 https://registry.npmjs.org/nouislider 3ms (cache hit)
62 silly fetch manifest plyr@^3.7.8
63 http fetch GET 200 https://registry.npmjs.org/plyr 2ms (cache hit)
64 silly fetch manifest star-rating.js@^4.3.0
65 http fetch GET 200 https://registry.npmjs.org/star-rating.js 2ms (cache hit)
66 silly fetch manifest tinymce@^6.4.2
67 http fetch GET 200 https://registry.npmjs.org/tinymce 2ms (cache hit)
68 silly fetch manifest tom-select@^2.2.2
69 http fetch GET 200 https://registry.npmjs.org/tom-select 2ms (cache hit)
70 silly fetch manifest bootstrap@~5.2.2
71 http fetch GET 200 https://registry.npmjs.org/bootstrap 2ms (cache hit)
72 silly fetch manifest resolve-url-loader@^5.0.0
73 http fetch GET 200 https://registry.npmjs.org/resolve-url-loader 2ms (cache hit)
74 silly fetch manifest sass@^1.56.1
75 http fetch GET 200 https://registry.npmjs.org/sass 4ms (cache hit)
76 silly placeDep ROOT @popperjs/core@2.11.7 OK for:  want: ^2.11.6
77 silly placeDep ROOT @tabler/core@1.0.0-beta19 OK for:  want: ^1.0.0-beta16
78 silly placeDep ROOT autosize@5.0.2 OK for:  want: ^5.0.1
79 silly placeDep ROOT bootstrap@5.2.3 OK for:  want: ~5.2.2
80 silly placeDep ROOT imask@6.6.1 OK for:  want: ^6.4.3
81 silly placeDep ROOT resolve-url-loader@5.0.0 OK for:  want: ^5.0.0
82 silly placeDep ROOT sass@1.62.1 OK for:  want: ^1.56.1
83 silly fetch manifest bootstrap@5.3.0-alpha3
84 silly fetch manifest adjust-sourcemap-loader@^4.0.0
85 silly fetch manifest convert-source-map@^1.7.0
86 silly fetch manifest loader-utils@^2.0.0
87 silly fetch manifest source-map@0.6.1
88 silly fetch manifest chokidar@>=3.0.0 <4.0.0
89 silly fetch manifest immutable@^4.0.0
90 http fetch GET 200 https://registry.npmjs.org/immutable 7ms (cache hit)
91 http fetch GET 200 https://registry.npmjs.org/source-map 9ms (cache hit)
92 http fetch GET 200 https://registry.npmjs.org/loader-utils 9ms (cache hit)
93 http fetch GET 200 https://registry.npmjs.org/convert-source-map 11ms (cache hit)
94 http fetch GET 200 https://registry.npmjs.org/adjust-sourcemap-loader 11ms (cache hit)
95 http fetch GET 200 https://registry.npmjs.org/chokidar 11ms (cache hit)
96 timing idealTree:#root Completed in 5157ms
97 timing idealTree:node_modules/@popperjs/core Completed in 0ms
98 silly fetch manifest @popperjs/core@^2.11.7
99 timing idealTree Completed in 5176ms
100 timing command:install Completed in 5181ms
101 verbose stack Error: could not resolve
101 verbose stack     at PlaceDep.failPeerConflict (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:546:25)
101 verbose stack     at PlaceDep.place (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:197:21)
101 verbose stack     at new PlaceDep (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:71:10)
101 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:965:31
101 verbose stack     at Array.map (<anonymous>)
101 verbose stack     at Arborist.[buildDepStep] (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:965:8)
101 verbose stack     at async Arborist.buildIdealTree (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:215:7)
101 verbose stack     at async Promise.all (index 1)
101 verbose stack     at async Arborist.reify (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/reify.js:152:5)
101 verbose stack     at async Install.exec (/usr/local/lib/node_modules/npm/lib/commands/install.js:157:5)
102 verbose cwd /Users/nibirahmed/Sites/nsu-app
103 verbose Darwin 22.3.0
104 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
105 verbose node v16.14.2
106 verbose npm  v8.5.0
107 error code ERESOLVE
108 error ERESOLVE could not resolve
109 error
110 error While resolving: [1m@tabler/core[22m@[1m1.0.0-beta19[22m
110 error Found: [1mautosize[22m@[1m5.0.2[22m[2m[22m
110 error [2mnode_modules/autosize[22m
110 error   [33mdev[39m [1mautosize[22m@"[1m^5.0.1[22m" from the root project
110 error
110 error Could not resolve dependency:
110 error peerOptional [1mautosize[22m@"[1m^6.0.1[22m" from [1m@tabler/core[22m@[1m1.0.0-beta19[22m[2m[22m
110 error [2mnode_modules/@tabler/core[22m
110 error   [33mdev[39m [1m@tabler/core[22m@"[1m^1.0.0-beta16[22m" from the root project
110 error
110 error Conflicting peer dependency: [1mautosize[22m@[1m6.0.1[22m[2m[22m
110 error [2mnode_modules/autosize[22m
110 error   peerOptional [1mautosize[22m@"[1m^6.0.1[22m" from [1m@tabler/core[22m@[1m1.0.0-beta19[22m[2m[22m
110 error   [2mnode_modules/@tabler/core[22m
110 error     [33mdev[39m [1m@tabler/core[22m@"[1m^1.0.0-beta16[22m" from the root project
110 error
110 error Fix the upstream dependency conflict, or retry
110 error this command with --force, or --legacy-peer-deps
110 error to accept an incorrect (and potentially broken) dependency resolution.
110 error
110 error See /Users/nibirahmed/.npm/eresolve-report.txt for a full report.
111 verbose exit 1
112 timing npm Completed in 5314ms
113 verbose unfinished npm timer reify 1684558701372
114 verbose unfinished npm timer reify:loadTrees 1684558701376
115 verbose unfinished npm timer idealTree:buildDeps 1684558701393
116 verbose unfinished npm timer idealTree:node_modules/@tabler/core 1684558706551
117 verbose code 1
118 error A complete log of this run can be found in:
118 error     /Users/nibirahmed/.npm/_logs/2023-05-20T04_58_21_240Z-debug-0.log
`

--- a/src/Console/InstallCommand.php
+++ b/src/Console/InstallCommand.php
@@ -536,9 +536,9 @@ class InstallCommand extends Command
         // NPM Packages...
         $this->updateNodePackages(function ($packages) {
             $dependencies = [
-                "@tabler/core"       => "^1.0.0-beta16",
+                "@tabler/core"       => "^1.0.0-beta19",
                 "resolve-url-loader" => "^5.0.0",
-                "autosize"           => "^5.0.1",
+                "autosize"           => "^6.0.1",
                 "imask"              => "^6.4.3",
                 "sass"               => "^1.56.1",
                 "bootstrap"          => "~5.2.2",

